src/widgets/modalChildren/PatientEditModal.js

```diff
@@ -123,26 +113,24 @@ PatientEditModalComponent.propTypes = {
   surveyForm: PropTypes.object,
   nameNoteIsValid: PropTypes.bool,
   surveySchema: PropTypes.object,
-  onOpen: PropTypes.func.isRequired,
   onUpdateForm: PropTypes.func.isRequired,
 };
 
 const mergeProps = (stateProps, dispatchProps, ownProps) => {
   const { completedForm } = stateProps;
   const { onOpen, onSave, onSaveSurvey, ...otherDispatchProps } = dispatchProps;
-  const { patient, surveySchema } = ownProps;
-  const { id = generateUUID() } = patient ?? {};
+  const { surveySchema } = ownProps;
+
   const onSaveForm = () => {
-    onSave({ id, ...completedForm });
-    if (surveySchema) onSaveSurvey(id);
+    onSave(completedForm);
+    if (surveySchema) onSaveSurvey();
```

Comment:
don't need to use these ids now because `dispensingPage` callbacks are creating a default `patient` which has an `id`.